tests/dialects/test_bigquery.py

```diff
@@ -251,6 +251,7 @@ def test_bigquery(self):
                 "duckdb": "SELECT STRFTIME(CAST('2023-12-25' AS DATE), '%Y%m%d')",
             },
         )
+        self.validate_all("SELECT COUNTIF(x)", write={"duckdb": "SELECT COUNT_IF(x)"})
```

Comment:
add a bigquery test here "bigquery": ... and make sure that passes, i assume it won't because you haven't added a rename_func for bigquery

Reply:
just did.  the test still passes